docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Since you already have tox, why not use it instead of having two make shim files, hence another level of indirection? Could do something like this:
```ini
[testenv:build-docs]
basepython = python3
description = Build The Docs
deps =
    Sphinx
changedir = docs
commands =
    # Build docs site:
    python -m sphinx \
      -j auto \
      -a -n -W \
      -b html --color \
      -d "{toxworkdir}/docs_doctree" \
      . "{toxworkdir}/docs_out"

    # Print out the output docs dir and a way to serve html:
    python -c\
    'import pathlib;\
    docs_dir = pathlib.Path(r"{toxworkdir}") / "docs_out";\
    index_file = docs_dir / "index.html";\
    print(f"\nDocumentation available under\
    `file://\{index_file\}`\n\nTo serve docs, use\
    `python3 -m http.server --directory \
    \N\{QUOTATION MARK\}\{docs_dir\}\N\{QUOTATION MARK\} 0`\n")'
```
_(https://github.com/cherrypy/cheroot/blob/8ff6998/tox.ini#L45-L61)_

Reply:
Oh, I didn't see that `tox.ini` is the last file in this PR.

Reply:
But if you already have it, I'd remove `Makefile` and `make.bat` as unnecessary.

Reply:
Hm, I don't think this matters much one way or the other. These makefiles are generated automatically by `sphinx-quickstart`, so people are kind of expecting them to be there. They are convenient for stuff like `make pdf`, though on the other hand this repo is eventually going to become just a backport for the standard library, so the documentation configuration doesn't matter much.

I think on balance I'll just keep them as "not hurting anything", then remove them if anyone reports a bug :stuck_out_tongue: